docs/products/postgresql/howto/postgresql-debezium-cdc.rst

```diff
@@ -0,0 +1,192 @@
+-----------------------------------------------------------------------------------
+
+Per the Debezium docs, there are two reasons why this can happen (copied
+verbatim from the `Debezium
+docs <https://debezium.io/documentation/reference/1.5/connectors/postgresql.html#postgresql-wal-disk-space>`__
+):
+
+#. *There are many updates in a database that is being tracked but only
+   a tiny number of updates are related to the table(s) and schema(s)
+   for which the connector is capturing changes. This situation can be
+   easily solved with periodic heartbeat events. Set the
+   ``heartbeat.interval.ms`` connector configuration property.*
+
+#. *The PostgreSQL instance contains multiple databases and one of them
+   is a high-traffic database. Debezium captures changes in another
+   database that is low-traffic in comparison to the other database.
+   Debezium then cannot confirm the LSN
+   (*\ ```confirmed_flush_lsn`` <https://www.postgresql.org/docs/13/view-pg-replication-slots.html>`__\ *)
+   as replication slots work per-database and Debezium is not invoked.
+   As WAL is shared by all databases, the amount used tends to grow
+   until an event is emitted by the database for which Debezium is
+   capturing changes.*
+
+During basic testing, this has been observed to happen in 2 scenarios:
+
+#. The table(s) which the Debezium connector is tracking has not had any
+   changes, and heartbeats are not enabled in the connector.
+
+#. The table(s) which the Debezium connector is tracking has not had any
+   changes, heartbeats are enabled (via
+   ```heartbeat.interval.ms`` <https://debezium.io/documentation/reference/1.5/connectors/postgresql.html#postgresql-property-heartbeat-interval-ms>`__
+   and
+   ```heartbeat.action.query`` <https://debezium.io/documentation/reference/1.5/connectors/postgresql.html#postgresql-property-heartbeat-action-query>`__
+   ), but the connector is not sending the heartbeat. Debezium not
+   sending the heartbeat is a known bug reported in
+   `DBZ-3746 <https://issues.redhat.com/browse/DBZ-3746>`__ .
+
+.. _h_7415120456:
+
+Clearing the replication lag
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To clear the lag, simply resume the database activities (assuming you
+have paused all traffic to the database) or make some changes to any
+table in the database in which Debezium is capturing changes to invoke
+the replication slot (such as the heartbeat table). Debezium would then
+confirm the latest LSN and allow the database to reclaim the WAL space.
+
+.. _h_b915a23266:
+
+Ensuring Debezium gracefully survives a PostgreSQL node replacement
+-----------------------------------------------------------------
+
+To prevent data loss (Debezium missing change events) in the event of a
+PostgreSQL node replacement and ensure the automatic recovery of the
+connector, here are several guidelines which you can follow.
+
+.. _h_a711a06482:
+
+Immediate solutions
```

Comment:
What is this immediate solution and why should one use it? I believe the title needs a bit more context